### PR TITLE
Small UI Fixes

### PR DESCRIPTION
**Pull request by rottabonus**
-Small device fixes for statusmessage
-From chat able to navigate to MentorCard
-Indication of used characters in statusmessage
-Fix translation

### CODE REVIEW

**Review by ahto — DISMISSED** [review]

File: src/Screens/Main/Chat/Title.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ type OwnProps = {
   buddyId: string;
   onLayout: (e: RN.LayoutChangeEvent) => void | undefined;
   openDropdown: () => void | undefined;
+  goToMentorCard: () => void;
```
Comment (pheis) [inline src/Screens/Main/Chat/Title.tsx:28]: would add this as void | undef for sake of consistency. no biggie.

File: src/Screens/Main/Chat/index.tsx [hunk 2]
```diff
@@ -17,18 +18,35 @@ import { Dialog, DialogProps } from '../../components/Dialog';
...
 
+  const buddyId = navigation.getParam('buddyId');
+
+  const mentor = ReactRedux.useSelector(mentorState.getMentorByUserId(buddyId));
+
+  const goToMentorCard = () => {
+    if (mentor) {
+      navigation.navigate('Main/MentorCardExpanded', {
+        mentor,
+        fromChat: true,
```
Comment (pheis) [inline src/Screens/Main/Chat/index.tsx:45]: "didNavigateFromChat" would make it more certain that is is boolean. From chat could be Id or something. this is just fine as it is tho.

File: src/Screens/components/MentorTitle.tsx [hunk 3]
```diff
@@ -65,10 +67,14 @@ const MentorTitle: React.FC<Props> = ({
...
+              <RN.Text>{age}</RN.Text>
               <Message id={'components.mentorCard.yearsAbbrev'} />
-              <RN.Text>{' | '}</RN.Text>
-              <RN.Text>{region}</RN.Text>
+              {hasRegion && (
+                <>
+                  <RN.Text>{' | '}</RN.Text>
+                  <RN.Text>{region}</RN.Text>
+                </>
+              )}
```
Comment (pheis) [inline src/Screens/components/MentorTitle.tsx:77]: nice

**Review by pheis — DISMISSED** [review]
solid!

**Review by ahto — DISMISSED** [review]

**Review by ahto — APPROVED** [review]

**Pull request merged**